framework/src/org/checkerframework/framework/flow/CFAbstractTransfer.java

```diff
@@ -721,12 +732,41 @@ protected void addInformationFromPreconditions(S info,
 
         S info = in.getRegularStore();
         V rhsValue = in.getValueOfSubNode(rhs);
+
+        Receiver expr = FlowExpressions.internalReprOf(analysis.getTypeFactory(),
+                n.getTarget());
+
+        if (useJaifInference && !expr.containsUnknown()
+                && expr instanceof FieldAccess) {
+            // Updates .jaif file
+            ClassSymbol clazzSymbol = JaifInferenceUtils.getClassSymbol(analysis.
```

Comment:
I personally don't like breaking a line at a field/method access.  I would rather break the line at existing whitespace, or before the open parenthesis.  Also, lines 743 and 744 are inconsistent about whether the period appears before the line break or after it; please be consistent, for readability.